vraptor-site/content/en/docs/converters.html

```diff
@@ -41,6 +41,16 @@
 
 If you are using JDK 8, you can use the new `java.time` API available using [vraptor-java8](/en/docs/plugins#vraptor-java-8) plugin.
 
+You can also customize the Date pattern by implementing your own CustomDateGsonConverter and overriding the method `getPattern()`:
```

Comment:
What do you think about moving this doc to the end of this page, using the follow title?

> #Overriding custom JSON and XML converters

It sounds better to me, because only in that point we tell how to override VRaptor beans.